src/SMG/UserBundle/Entity/User.php

```diff
@@ -67,9 +67,13 @@ public function getPhoneNumber()
     }
 
     /**
+     * TODO: problem on Symfony client side when use 
+     * an already declared function name, need to
+     * investigate why.
```

Comment:
not descriptive enough :/ 

and i dont understand what it does with the client ?